Twitter.js

```diff
@@ -87,9 +87,21 @@ function scrape(doc, url) {
 	}
 	var date = ZU.xpathText(doc, '//div[contains(@class,"permalink-tweet-container")]//span[@class="metadata"]/span[1]');
 	if (date) {
-		var m = date.match(/(\d\d:\d\d)\s*-\s*(.*)/);
-		if (m) {
-			item.date = ZU.strToISO(m[2]) + "T" + m[1];
+		// e.g. 10:22 AM - 1 Feb 2018
+		var m = date.split('-');
+		if (m.length == 2) {
+			// times with AM
+			if (m[0].includes("AM")) m[0] = m[0].replace("AM", "").trim();
+			if (m[0].indexOf(":")==1) m[0] = "0" + m[0];
+			m[0].replace("12:", "00:");
```

Comment:
This changes 12:35 PM to 00:35 PM

Reply:
Ah, that should be only performed when the `AM` is found... Moreover, the `m[0] = ` is missing here...